packages/xs-extra/xapi-nbd.master/opam

```diff
@@ -24,7 +24,7 @@ depends: [
   "sexplib"
   "mirage-block-unix"
   "io-page"
-  "mirage"
+  "mirage" {< "3.0.0"}
```

Comment:
Don't think so, we should just remove the dependencies. I think they are only transitive deps.